### PR TITLE
fix: call participants tiles UI [WPB-10149]

### PR DESCRIPTION
**Pull request by saleniuk**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-10149" title="WPB-10149" target="_blank">BugWPB-10149</a>  [Android] Tiles in calling UI are cut of on last row
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

During a call with more then 8 participants, the last row of tiles is cut off.
User name can overlap the mute icon.
Some ongoing call elements don't match designs.

### Solutions

- Calculate height of tiles correctly.
- Fix layout of user name, "connecting" label and "mute" icon to make it all be shown properly.
- Update avatar composable to allow using it without any indications (status, legal hold) and thus to not have any unnecessary paddings. It allowed also to fix and make typing indicator to be like on designs.
- Make corners and borders to match designs ("active speaker" border, name and mute icon corners).

#### How to Test

Create a call with multiple participants and check the UI.

### Attachments (Optional)

These screenshots below are in high resolution, click and open to see in detail and compare.

| Before | After | Design |
| ----------- | ------------ | ------------ |
| tiles_before | tiles_after | tiles_design |
| call_screen_before | call_screen_after | call_sceen_design |
| typing_indicator_before | typing_indicator_after | typing_indicator_design |

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by alexandreferris — APPROVED** [review]
oooh they looking amazing now 🔥

**Review by yamilmedina — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/UsersTypingIndicator.kt [hunk 1]
```diff
@@ -120,16 +126,32 @@ fun UsersTypingIndicator(usersTyping: List<UIParticipant>) {
...
+        modifier = modifier,
+        horizontalArrangement = Arrangement.spacedBy(spacedBy)
+    ) {
         usersTyping.take(maxPreviewsDisplay).forEach { user ->
             UserProfileAvatar(
                 avatarData = user.avatarData,
-                size = dimensions().spacing16x,
-                padding = dimensions().spacing2x,
-                showStatusIndicator = false,
+                size = 16.dp,
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/UsersTypingIndicator.kt]:
```suggestion
                size = dimensions().spacing16x,
```

File: app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallScreen.kt [hunk 2]
```diff
@@ -499,8 +483,45 @@ fun PreviewOngoingCallScreen() = WireTheme {
...
+fun PreviewOngoingCallScreen_2Participants() = WireTheme {
+    PreviewOngoingCallContent(participants = buildPreviewParticipantsList(2))
+}
+
+@PreviewMultipleThemes
+@Composable
+fun PreviewOngoingCallScreen_8Participants() = WireTheme {
+    PreviewOngoingCallContent(participants = buildPreviewParticipantsList(8))
+}
+
```
Comment (yamilmedina) [inline app/src/main/kotlin/com/wire/android/ui/calling/ongoing/OngoingCallScreen.kt:503]: Thanks ! 👏

**Review by yamilmedina — COMMENTED** [review]
Just a small thing, a `dimension` that slipped, great work as always 👏

**Review by ohassine — APPROVED** [review]
Awesome! and thanks for the previews 🚀

**Pull request merged**